corehq/apps/app_manager/tests/test_bulk_app_translation.py

```diff
@@ -215,6 +231,20 @@ def test_missing_itext(self):
         except Exception as e:
             self.fail(e)
 
+    def test_bad_column_name(self):
+        self.upload_raw_excel_translations(self.upload_headers_bad_column,
+            self.upload_data,
+            expected_messages=[
+                u'Sheet "module1_form1" has less columns than expected. Sheet '
+                'will be processed but the following translations will be '
+                'unchanged: default_fra',
+                u'Sheet "module1_form1" has unrecognized columns. Sheet will '
+                'be processed but ignoring the following columns: default-fra',
+                u'App Translations Updated!'
```

Comment:
Yeah. I looked into doing that, but partially out of laziness I didn't.

To fix the other columns would need to be added here: https://github.com/dimagi/commcare-hq/blob/d08268fdefab7fe7da025723be96221e008f9269/corehq/apps/app_manager/translations.py#L191

And there'd need to be another check to make sure we don't warn on default-fra when they use default_fra and vice versa.

It seemed like more work than necessary to refactor this. Especially since vellum now defaults to using the same format as HQ.